### ISSUE TITLE
any way to turn off exception messages?

### ISSUE THREAD

**Issue by Phouter0499** [singhsidhukuldeep/request-boost#12]
`url_ske = "https://api.dictionaryapi.dev/api/v2/entries/en/{}"
urls = [url_ske.format(lemma) for lemma in ["bugler","hello","scaper"]]
results = boosted_requests(urls=urls, max_tries=1)`
the code above results in "AssertionError: Maximum number of attempts reached 1"
i think the reason is that free dictionary api returns a 404 response object whenever it fails to find any def for a word.
i could just turn off verbose but that turns off progress as well :(
please fix thanks.

**Comment by Phouter0499** [comment]
update:
turning off verbose doesn't turn off display of exceptions

**Comment by singhsidhukuldeep** [comment]
Makes sense, I will add an option for what to do if it fails, eg return some value or raise an assertion error

Can you contribute to this, or should I code it?

**Comment by Phouter0499** [comment]
problem occurs around line 81 in the __init__.py file
heres my ducktape solution:
(line 80 to 82)
```
assert (num_tries < self.max_tries), f"Maximum number of attempts reached {self.max_tries} for {content}"
```
to
```
if num_tries == self.max_tries:
                        break
```
it works in my juypter notebook
here's the test code image:
![image](https://github.com/singhsidhukuldeep/request-boost/assets/124970057/aa55622f-6241-4f59-94d3-c98efbd3c9e0)

**Comment by singhsidhukuldeep** [comment]
Solved in ver 0.8
